dev/keep_updating_pt_user_stats.rb

```diff
@@ -0,0 +1,10 @@
+interrupted = false
```

Comment:
This script looks like a daemon? If so we should put this in to daemons folder.

Reply:
Anyway, dev scripts are not designed to be run on production.

Reply:
It's only for dev, so should stay in dev folder

Reply:
Instead, please add ```raise "Dev script can't be executed on production" if Rails.env.production?``` line (it's been added in a meantime to other dev scripts in this story https://github.com/brianlong/www.validators.app/pull/945/files, so let's keep it consistant)